openapi/v2/support-notifications.yaml

```diff
@@ -15,93 +15,19 @@ components:
-          description: "Categorizes the notification. Must be one of three values: SECURITY, HW_HEALTH, SW_HEALTH"
-        content:
-          description: "The actual content to be sent as the body of the notification."
-          type: string
-        contentType:
-          description: "Indicates the MIME type/Content-type of the notification's content."
-          type: string
-        description:
-          description: "An optional description of the notification's intent."
-          type: string
-        labels:
-          description: "Arbitrary labels that can be applied to notifications for further categorization or identification."
-          type: array
-          items:
-            type: string
-        sender:
-          description: "Identifies the sender of a notification, usually an email address."
-          type: string
-        severity:
-          description: "Indicates the level of severity for the notification. Current accepted values include: NORMAL, CRITICAL"
-          type: string
-        slug:
-          description: "Effectively a name or key that labels the notification."
-          type: string
-        status:
-          description: "A status indicating the current processing status of the notification. This should be set to NEW when adding a notification."
-          type: string
+        notification:
+          $ref: '#/components/schemas/Notification'
       required:
-      - category
-      - content
-      - sender
-      - severity
-      - slug
-    AddNotificationResponse:
-      allOf:
-      - $ref: '#/components/schemas/BaseResponse'
-      type: object
-      properties:
-        id:
-          type: string
-          format: uuid
-        slug:
-          type: string
+        - notification
     AddSubscriptionRequest:
       allOf:
       - $ref: '#/components/schemas/BaseRequest'
       type: object
       properties:
-        categories:
-          description: "Links the subscription to one or more categories of notification. Must be one of three values: SECURITY, HW_HEALTH, SW_HEALTH"
-          type: array
-          items:
-            type: string
-        channels:
-          description: "Indicates how this subscription is capable of receiving notifications."
-          type: array
-          items:
-            $ref: '#/components/schemas/Channel'
-        description:
-          description: "An optional description of the subscription's intent."
-          type: string
-        labels:
-          description: "Arbitrary labels that can be applied to the subscription for further categorization or identification."
-          type: array
-          items:
-            type: string
-        receiver:
-          description: "The name of the party interested in the notification."
-          type: string
-        slug:
-          description: "Links the subscription to the name or key that labels the notification of interest."
-          type: string
+        subscription:
```

Comment:
Missing top level description for `AddSubscriptionRequest`

Reply:
Added.